### DISCUSSION TITLE
global_coherence.plot() parameters

### DISCUSSION THREAD

**Discussion by thelightonmyway** [LinkedEarth/Pyleoclim_util#628] [Q&A]
Hi,

I used `global_coherence` recently and I have some question when I plot it. Please look at my plot.

![1d0ce9172e76b1a1490b0456cb39d8c](https://github.com/user-attachments/assets/9a0ba09b-562e-4218-afd8-e781f7acf3a7)

What I expect is:
a) sets the line width for the significant test;
b) modify the label of the legend, especially the significance test.

In addition, I'm interested in why the significant test at the beginning of the `xaxis` is so low and steep rise  along `xaxis`, like where I framed it.

Any application will be appreciated!

Yang.

**Comment by CommonClimate** [comment]
Hi Yang, it seems like all your questions are matplotlib questions. If you export your graphic as: `fig, ax = coherence_object.plot()` then you can go into `ax` and manipulate it. Some examples of Matplotlib customization are [here](http://linked.earth/PyleoTutorials/notebooks/L1_figures_with_multiple_panels.html) and in [PaleoBooks](http://linked.earth/PaleoBooks/)

**Reply by thelightonmyway** [reply]
OK, I will try it. And I'm still confused about the red noise which suddenly dropped in the high period region and I marked it in the red box, could you please tell me if it appears anomaly?

**Comment by CommonClimate** [comment]
This is at the very edge of the frequency axis, very likely affected by edge effects, so I would dismiss this part entirely (i.e. limit the xaxis to values where the red noise benchmark makes sense).
@alexkjames is there a GlobalCoherence equivalent to the wavelet cone of influence? I wonder how we can denote that in a useful way (e.g. what fraction of the values were under the COI before time-summation)?

**Reply by alexkjames** [reply]
> This is at the very edge of the frequency axis, very likely affected by edge effects, so I would dismiss this part entirely (i.e. limit the xaxis to values where the red noise benchmark makes sense). @alexkjames is there a GlobalCoherence equivalent to the wavelet cone of influence? I wonder how we can denote that in a useful way (e.g. what fraction of the values were under the COI before time-summation)?

That's an interesting point, certainly I could add a `% of the time axis used` option to the plotting method so that the robustness of results could more easily be verified

**Reply by thelightonmyway** [reply]
I will take your advice. Thanks for your reply!
